pages/configuration/multinode_setup.rst

```diff
@@ -57,9 +57,15 @@ When you secure your Elasticsearch with `User Authentication <https://www.elasti
 Graylog Multi-node
 ==================
 
-After the installation of Graylog, you should take care that only one Graylog node is configured to be master with the configuration setting ``is_master = true``.
+After the installation of Graylog, you should take care that only one Graylog node is configured to be master with the
+configuration setting ``is_master = true``.
 
-The HTTP settings configured in ``http_bind_address`` (or ``http_publish_uri``) must be accessable for all Graylog nodes of the cluster.
+The HTTP settings configured in ``http_bind_address`` (or ``http_publish_uri``) must be accessable for all Graylog nodes
```

Comment:
the correct statement here is:

The ``http_bind_address`` configured address need to be reachable by all Graylog nodes in the cluster. Same for the ``http_publish_uri`` which is mostly the the ``http_bind_address`` prefixed with the protocol and a closing ``/``.